packages/embed-react/src/components/make-button-component.tsx

```diff
@@ -26,7 +27,7 @@ const emptyEmbed: GenericEmbed = {
 }
 
 function makeButtonComponent<T>(createFn: CreateFn<T>, cssFilename: string) {
-  return ({ id, children, style = {}, className = '', ...props }: ButtonComponentProps<T>) => {
+  return ({ id, children, as = 'button', style = {}, className = '', ...props }: ButtonComponentProps<T>) => {
```

Comment:
```suggestion
  return ({ id, children, as = 'span', style = {}, className = '', ...props }: ButtonComponentProps<T>) => {
```
Maybe we should even fallback to `<span />` or `<div />` ?

Reply:
I think we need to fallback to `<button>` otherwise exiting implementations will be affected.